spec/support/hanami_commands.rb

```diff
@@ -15,6 +15,7 @@ def rackup(args = {}, &blk)
       end
 
       def server(args = {}, &blk)
+        args[:port] ||= rand(1000..9000)
```

Comment:
It'd actually be ideal to use only prime numbers between 1025 and 9999 if there's a good way to code that.  Many folks may have other development servers running on ports like 3000, 5000, 8000, 8100, et cetera.  It's very unlikely the prime numbers will be used already, because it's not in human nature.